### PR TITLE
VIH-10376 New users seeing you do not have a hearing page

### PR DESCRIPTION
**Pull request by will-craig**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/VIH-10376

### Change description ###
Changes get conferences for individual and host endpoints, to get the hearing ids from booking api first then query the conferences with the hearing ids, rather than the username

### CODE REVIEW

File: VideoWeb/VideoWeb/Mappings/BookingForHostResponseMapper.cs [hunk 1]
```diff
@@ -20,23 +20,20 @@ public BookingForHostResponseMapper(IMapTo<ParticipantForHostResponse, VideoWeb.
...
-            ScheduledDateTime = booking.ScheduledDateTime,
-            NumberOfEndpoints = booking.Endpoints.Count,
-            HearingVenueIsScottish = booking.IsHearingVenueScottish,
-            Status = Enum.Parse<ConferenceStatus>(conference.Status.ToString()),
-            ClosedDateTime = conference.ClosedDateTime,
-            Participants =
-                conference.Participants.Select(_participantForHostResponseMapper.Map)
-                    .ToList() // need to use participant list from video api to include QL users
-        };
+        var dto = new Contract.Responses.ConferenceForHostResponse();
```
Comment (oliver-scott) [inline VideoWeb/VideoWeb/Mappings/BookingForHostResponseMapper.cs:23]: are these changes needed? Probably best to use the object initialiser if we can

File: charts/vh-video-web/values.dev.template.yaml [hunk 2]
```diff
@@ -10,4 +10,5 @@ java:
         EJUDAD__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
         EJUDAD__REDIRECTURI: https://${SERVICE_FQDN}/home
         DOM1__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
-        DOM1__REDIRECTURI: https://${SERVICE_FQDN}/home
\ No newline at end of file
+        DOM1__REDIRECTURI: https://${SERVICE_FQDN}/home        
+        VHSERVICES__VIDEOAPIURL: https://vh-video-api-pr-626.dev.platform.hmcts.net/
```
Comment (oliver-scott) [inline charts/vh-video-web/values.dev.template.yaml:14]: just a reminder to change before the merge

File: VideoWeb/VideoWeb.Common/VideoWeb.Common.csproj [hunk 3]
```diff
@@ -20,7 +20,7 @@
     </PackageReference>
     <PackageReference Include="StackExchange.Redis" Version="2.6.111" />
     <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.6.0" />
-    <PackageReference Include="VideoApi.Client" Version="1.48.7" />
+    <PackageReference Include="VideoApi.Client" Version="1.50.1-pr-0626-0005" />
```
Comment (oliver-scott) [inline VideoWeb/VideoWeb.Common/VideoWeb.Common.csproj:23]: just a reminder to change before the merge

**Review by oliver-scott — APPROVED** [review]
nicely done. Just a minor observation added, not a dealbreaker though

File: VideoWeb/VideoWeb/Mappings/BookingForHostResponseMapper.cs (hunk 1 above)
Comment (will-craig) [reply VideoWeb/VideoWeb/Mappings/BookingForHostResponseMapper.cs:23]: I actually prefer it this way, because if it throws an exception you can see immediately which line it was on, and which property has the null ref, rather than it just pointing to line 23, and they you have to work it out. Would save debug time if we didnt use object initialisers everywhere

Comment (oliver-scott) [reply VideoWeb/VideoWeb/Mappings/BookingForHostResponseMapper.cs:23]: understandable
